src/utils/components/withMatchMedia.jsx

```diff
@@ -44,7 +44,7 @@ const withMatchMedia = WrappedComponent => {
 			super(props);
 
 			this.state = {
-				media: {},
+				media: props.initialMedia || {},
```

Comment:
when would this be instantiated with/without props.initialMedia?

Reply:
I created a wrapper component that passes initial media here: https://github.com/meetup/meetup-web-platform/pull/423/files#diff-e8b9f22f4184c5ccc6b38a7e5e3e7799R24
But this is being used in a lot of places without the new wrapper component, so if it it's used on its own, it will not have initial media.